src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+    Animal.alive.push(this);
```

Comment:
Adding the `Herbivore` instance to the `Animal.alive` array here is correct, but ensure that this logic is consistent across all animal types. Consider adding a method in the `Animal` class to handle this logic to avoid duplication.